libs/wire-api/src/Wire/API/Team.hs

```diff
@@ -107,15 +107,18 @@ newTeam tid uid nme ico tb = Team tid uid nme ico Nothing tb DefaultIcon
 
 instance ToSchema Team where
   schema =
-    object "Team" $
+    objectWithDocModifier "Team" desc $
       Team
         <$> _teamId .= field "id" schema
         <*> _teamCreator .= field "creator" schema
         <*> _teamName .= field "name" schema
         <*> _teamIcon .= field "icon" schema
         <*> _teamIconKey .= maybe_ (optField "icon_key" schema)
-        <*> _teamBinding .= (fromMaybe Binding <$> optField "binding" schema)
+        <*> _teamBinding .= (fromMaybe Binding <$> optFieldWithDocModifier "binding" bindingDesc schema)
         <*> _teamSplashScreen .= (fromMaybe DefaultIcon <$> optField "splash_screen" schema)
+    where
+      desc = description ?~ "`binding` is deprecated, and should be ignored. The non-binding teams API is not used (and will not be supported from API version V4 onwards), and `binding` should always be `true`."
+      bindingDesc = description ?~ "Deprecated, please ignore."
```

Comment:
```suggestion
      desc = description ?~ "`binding` is deprecated, and should be ignored. The non-binding teams API is not used (and will not be supported from API version V4 onwards), and `binding` will always be `true`."
      bindingDesc = description ?~ "Deprecated, please ignore."
```

why the `desc`, and not just the `bindingDesc` with all the information near what it's about?

Reply:
Because it is easier to see. To see the field description, you need to scroll down and click on `Model` in the swagger UI.